torch/csrc/api/src/nn/modules/loss.cpp

```diff
@@ -48,6 +48,28 @@ Tensor CosineEmbeddingLossImpl::forward(
     const Tensor& target) {
   return F::cosine_embedding_loss(input1, input2, target, options);
 }
+// ============================================================================
+
+MultiLabelSoftMarginLossImpl::MultiLabelSoftMarginLossImpl(
+    const torch::nn::MultiLabelSoftMarginLossOptions& options_)
+    : options(options_) {}
+
+void MultiLabelSoftMarginLossImpl::pretty_print(std::ostream& stream) const {
+  stream << "torch::nn::MultiLabelSoftMarginLoss(weight=" << options.weight()
+    << ", reduction=" << options.reduction() << ")";
```

Comment:
It would be awesome to add a test for the `pretty_print` as well. :D Thanks a lot for your help @CarMiranda!